__tests__/app.test.js

```diff
@@ -194,6 +194,48 @@ describe('/api/articles', () => {
     })
 })
 
+describe('/api/articles?topic', () => {
+    test('GET 200: Should return articles filtered by topic when passed an existing topic', () => {
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then((response) => {
+            const { articles } = response.body
+            articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    article_id: expect.any(Number),
+                    title: expect.any(String),
+                    topic: expect.any(String),
```

Comment:
I know you have a second assertion for this, but your topic should be defined here if you're filtering by it...